att.go

```diff
@@ -0,0 +1,188 @@
+//go:build nina || nano_rp2040
+
+package bluetooth
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"errors"
+	"time"
+)
+
+const (
+	ATT_CID = 0x0004
+	BLE_CTL = 0x0008
+
+	ATT_OP_ERROR              = 0x01
+	ATT_OP_MTU_REQ            = 0x02
+	ATT_OP_MTU_RESP           = 0x03
+	ATT_OP_FIND_INFO_REQ      = 0x04
+	ATT_OP_FIND_INFO_RESP     = 0x05
+	ATT_OP_FIND_BY_TYPE_REQ   = 0x06
+	ATT_OP_FIND_BY_TYPE_RESP  = 0x07
+	ATT_OP_READ_BY_TYPE_REQ   = 0x08
+	ATT_OP_READ_BY_TYPE_RESP  = 0x09
+	ATT_OP_READ_REQ           = 0x0a
+	ATT_OP_READ_RESP          = 0x0b
+	ATT_OP_READ_BLOB_REQ      = 0x0c
+	ATT_OP_READ_BLOB_RESP     = 0x0d
+	ATT_OP_READ_MULTI_REQ     = 0x0e
+	ATT_OP_READ_MULTI_RESP    = 0x0f
+	ATT_OP_READ_BY_GROUP_REQ  = 0x10
+	ATT_OP_READ_BY_GROUP_RESP = 0x11
+	ATT_OP_WRITE_REQ          = 0x12
+	ATT_OP_WRITE_RESP         = 0x13
+	ATT_OP_WRITE_CMD          = 0x52
+	ATT_OP_PREP_WRITE_REQ     = 0x16
+	ATT_OP_PREP_WRITE_RESP    = 0x17
+	ATT_OP_EXEC_WRITE_REQ     = 0x18
+	ATT_OP_EXEC_WRITE_RESP    = 0x19
+	ATT_OP_HANDLE_NOTIFY      = 0x1b
+	ATT_OP_HANDLE_IND         = 0x1d
+	ATT_OP_HANDLE_CNF         = 0x1e
+	ATT_OP_SIGNED_WRITE_CMD   = 0xd2
+
+	GATT_UNKNOWN_UUID        = 0x0000
+	GATT_SERVICE_UUID        = 0x2800
+	GATT_CHARACTERISTIC_UUID = 0x2803
+	GATT_DESCRIPTOR_UUID     = 0x2900
+)
```

Comment:
These constants seem to be internal, I don't think they need to be exposed?

Reply:
Updated.